wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -132,12 +144,20 @@ def is_alive(self) -> bool:
             SchedulerState.COMPLETED,
             SchedulerState.FAILED,
             SchedulerState.STOPPED,
+            SchedulerState.CANCELLED,
         ]:
             return False
         return True
 
     def start(self) -> None:
         """Start a scheduler, confirms prerequisites, begins execution loop."""
+        if not self.is_alive():
+            wandb.termerror(
+                f"{LOG_PREFIX}Sweep already stopped or cancelled! Exiting..."
```

Comment:
When does a sweep enter the failed state? Shouldn't we tell users that is another reason it could be dead? Same with completed?

Reply:
Well this would only happen when `start` is called, which is before it can even fail (in theory). Ill change the print to reflect the real state, but i'm pretty sure this covers nearly all cases. This case only happens when a user queues a launch sweep scheduler, and then cancels it in the UI before a launch agent can pick it up.